gravitee-apim-console-webui/src/portal/customization/top-bar/menu-link-edit/menu-link-edit.component.html

```diff
@@ -0,0 +1,49 @@
+<!--
+
+    Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<portal-header title="Top Bar" subtitle="Customize the Developer Portal top bar." />
+
+<div class="back-button">
+  <a mat-button [routerLink]="'../'"><mat-icon svgIcon="gio:arrow-left"></mat-icon>Top Bar Menu</a>
+</div>
+
+<mat-card class="menu-link-edit">
+  <mat-card-title class="menu-link-edit__title">
+    <h3>
+      Edit Link <span class="gio-badge-neutral">{{ initialValue ? toReadableMenuLinkType(initialValue.type) : '' }}</span>
+    </h3>
+  </mat-card-title>
+  <form *ngIf="menuLinkEditForm" [formGroup]="menuLinkEditForm">
+    <mat-card-content class="menu-link-edit__content">
+      <div class="menu-link-edit__content__title">
+        <span class="subtitle">Details</span>
+        <span class="description">Edit the link details that will be displayed in the top menu bar</span>
+      </div>
+      <mat-form-field>
+        <mat-label>Name</mat-label>
+        <input matInput formControlName="name" required />
+        <mat-error *ngIf="menuLinkEditForm.controls.name.hasError('required')">Name is required.</mat-error>
```

Comment:
If in standalone, using @if is preferred --> @if is integrated directly into the Angular framework when NgIf needs to be imported. Also, control flows are the future 👽 , so potentially *ngIf will not be supported in future Angular versions.